src/features/people/PeopleDetails/peopleDetailsSlice.js

```diff
@@ -0,0 +1,35 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const peopleDetailsSlice = createSlice({
+    name: "personDetails",
+    initialState: {
+        personId: 0,
+        details: {},
+        status: "loading"
+    },
+    reducers: {
+        fetchDataSuccess: (state, { payload }) => {
+            state.details = payload.details;
+            state.status = "success";
+        },
+        fetchDataError: (state) => {
+            state.status = "error";
+        },
+        getDetailsForPerson: (state, { payload }) => {
+            state.status = "loading";
+            state.personId = payload.personId;
+        },
+    },
+});
+export const {
+    fetchDataSuccess,
+    getDetailsForPerson,
+    fetchDataError,
+} = peopleDetailsSlice.actions;
+
+export const selectPersonState = (state) => state.personDetails;
```

Comment:
`const selectPersonState = (state) => state.peopleDetails;
`
PS: you don't need to export that selector as it is only used in Slice

Reply:
Thanks a lot for your observation and suggestion! 🙏 I appreciate your input. I've made the adjustment and removed the unnecessary export for `selectPersonState`.
Feel free to share more insights if you have them! 😊